README.md

```diff
@@ -0,0 +1,89 @@
+[![PyPI version](https://img.shields.io/pypi/v/ansible-core.svg)](https://pypi.org/project/ansible-core)
```

Comment:
Could we have the detached links style kept?

This could be implemented as follows:
```suggestion
[![pypi-version-badge-image]][pypi-page-link]

[pypi-page-link]: https://pypi.org/project/ansible-core
[pypi-version-badge-image]: https://img.shields.io/pypi/v/ansible-core.svg
```

Also, the detached links can be all listed at the bottom of the document, keeping the plain-text bit easier to consume as plain text.

Reply:
It doesn't seem like this adds any real readability, and makes the resultant doc a lot more verbose. I could see it if we were reusing the links, but since these are all one-shot, I don't see the benefit to the added abstraction. eg, the linked 8 lines below turns into:

```
[![PyPI Version badge]][PyPI version link]
[![Docs badge]][Docs link]
[![Chat badge]][Chat link]
[![Build Status badge]][Build Status link]
[![Ansible Code of Conduct badge]][Ansible Code of Conduct link]
[![Ansible Mailing Lists badge]][Ansible Mailing Lists link]
[![Repository License badge]][Repository License link]
[![Ansible CII Best Practices Certification badge]][Ansible CII Best Practices Certification link]

[PyPI version badge]: https://img.shields.io/pypi/v/ansible-core.svg
[PyPI version link]: https://pypi.org/project/ansible-core
[Docs badge]: https://img.shields.io/badge/docs-latest-brightgreen.svg
[Docs link]: https://docs.ansible.com/ansible/latest/
[Chat badge]: https://img.shields.io/badge/chat-IRC-brightgreen.svg
[Chat link]: https://docs.ansible.com/ansible/latest/community/communication.html
[Build Status badge]: https://dev.azure.com/ansible/ansible/_apis/build/status/CI?branchName=devel
[Build Status link]: https://dev.azure.com/ansible/ansible/_build/latest?definitionId=20&branchName=devel
[Ansible Code of Conduct badge]: https://img.shields.io/badge/code%20of%20conduct-Ansible-silver.svg
[Ansible Code of Conduct link]: https://docs.ansible.com/ansible/latest/community/code_of_conduct.html
[Ansible Mailing Lists badge]: https://img.shields.io/badge/mailing%20lists-Ansible-orange.svg
[Ansible Mailing Lists link]: https://docs.ansible.com/ansible/latest/community/communication.html#mailing-list-information
[Repository License badge]: https://img.shields.io/badge/license-GPL%20v3.0-brightgreen.svg
[Repository License link]: COPYING
[Ansible CII Best Practices Certification badge]: https://bestpractices.coreinfrastructure.org/projects/2372/badge
[Ansible CII Best Practices Certification link]: https://bestpractices.coreinfrastructure.org/projects/2372
```

This doesn't feel like an improvement to me...

Reply:
@webknjaz Thanks for pointing this out. I wasn't aware that Markdown supported link labels.

However, I agree with @nitzmahone that for this use case it seems overly verbose and repetitive. On a side note, I did notice that the link references can contain spaces, and are used for the image alt text. That could come in handy elsewhere.